frontend/src/components/Chatbox.vue

```diff
@@ -29,6 +29,15 @@ export default {
     },
 
     methods: {
+        colourMessage(message) {
```

Comment:
This could probably be done with a ternary expression like `message.fromUser ? "text-red-900" : "text-red-800"`, also, doesn't this just make the chat message a different shade of red depending who it's from? Don't we want something like blue for the current user, and red for opponents?

Reply:
ternary expression wont look good since ive added a lot more css now. If you insist, ill change it.

Reply:
> ternary expression wont look good since ive added a lot more css now. If you insist, ill change it.

No, there's no problem with the if statement. It was just an idea since in the initial PR there was only a single class.